### PR TITLE
Update CPU key metrics template

### PR DESCRIPTION
**Pull request by acearun**
Updates the template to have:
1. Overview CPU heatmap
2. Detailed grid
3. Max CPU by region
4. Azure resource health heatmap

image

### CODE REVIEW

File: Workbooks/Virtual Machines/At-scale Metrics/Key Metrics.workbook [hunk 1]
```diff
@@ -171,218 +167,857 @@
...
-            "splitBy": null
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Resource health",
+            "subTarget": "Health",
+            "style": "link"
+          }
+        ]
+      },
+      "name": "links - 6"
```
Comment (yves-chan) [inline Workbooks/Virtual Machines/At-scale Metrics/Key Metrics.workbook]: nit: could change group name to "Link tabs"?

File: Workbooks/Virtual Machines/At-scale Metrics/Key Metrics.workbook [hunk 2]
```diff
@@ -171,218 +167,857 @@
...
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "selectedTab",
+        "comparison": "isEqualTo",
+        "value": "Health"
       },
-      "name": "text - 4"
+      "name": "Overview - Copy"
```
Comment (yves-chan) [inline Workbooks/Virtual Machines/At-scale Metrics/Key Metrics.workbook]: rename to "Health"?

Comment (acearun) [reply Workbooks/Virtual Machines/At-scale Metrics/Key Metrics.workbook]: Fixed

File: Workbooks/Virtual Machines/At-scale Metrics/Key Metrics.workbook (hunk 1 above)
Comment (acearun) [reply Workbooks/Virtual Machines/At-scale Metrics/Key Metrics.workbook]: Fixed

**Pull request merged**